src/pyhf/infer/mle.py

```diff
@@ -40,7 +40,7 @@ def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
         >>> bestfit_pars
         array([0.        , 1.0030512 , 0.96266961])
         >>> twice_nll
-        array([24.98393521])
+        array(24.98393521)
```

Comment:
If you print `twice_nll` you get the float you expect

```
>>> import pyhf
>>> model = pyhf.simplemodels.hepdata_like(signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0])
>>> observations = [51, 48]
>>> data = observations + model.config.auxdata
>>> bestfit_pars, twice_nll = pyhf.infer.mle.fit(data, model, return_fitted_val=True)
>>> print(f"twice_nll is {twice_nll} and of type {type(twice_nll)} with shape {pyhf.tensorlib.shape(twice_nll)}")
twice_nll is 24.98393521454011 and of type <class 'numpy.ndarray'> with shape ()
```

but a 0-d tensor reprs as a tensor still

Reply:
It would be nice to have this as a test for `pyhf.infer.mle.fit` to at least check the shape of the output?

Reply:
@kratsg These were added to `test_infer.py`